python/cog/server/runner.py

```diff
@@ -11,7 +11,7 @@
 import structlog
 from attrs import define
 from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry  # type: ignore
+from urllib3.util.retry import Retry
```

Comment:
TIL. This is wild.

https://github.com/psf/requests/blob/0e322af87745eff34caffe4df68456ebc20d9068/src/requests/packages.py#L6